src/jailer/src/cgroup.rs

```diff
@@ -48,11 +43,12 @@ impl CgroupBuilder {
             version: ver,
             hierarchies: HashMap::new(),
             mount_points: Vec::new(),
+            proc_mounts_path,
         };
 
         // search PROC_MOUNTS for cgroup mount points
-        let f = File::open(PROC_MOUNTS)
-            .map_err(|err| JailerError::FileOpen(PathBuf::from(PROC_MOUNTS), err))?;
+        let f = File::open(b.proc_mounts_path)
+            .map_err(|err| JailerError::FileOpen(PathBuf::from(&b.proc_mounts_path), err))?;
```

Comment:
nitpicking, but `&` can be removed.

```suggestion
            .map_err(|err| JailerError::FileOpen(PathBuf::from(b.proc_mounts_path), err))?;
```

Reply:
fixed in e77496a63070784ab99d69f438c221555d99b990